src/components/logo/logo.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Logo = ({isLogoLight}) => (
+  <div className="logo">
+    <a className={isLogoLight ? `logo__link logo__link--light` : `logo__link`}>
```

Comment:
```suggestion
    <a className={`logo__link ${isLogoLight ? ` logo__link--light` : ``}`}>
```

ну или что-то типа того) ну или еще можешь использовать npm classnames, ооочень хорошая штука. 

`className={classnames(`logo__link`, isLogoLight &&  logo__link--light`)}`